spec/requests/api/moves_controller_role_access_spec.rb

```diff
@@ -81,6 +81,13 @@
     end
 
     context 'when successful' do
+      before do
```

Comment:
This block isn't being tested / isn't relevant to the test? is a test missing or is this in the wrong place?

Reply:
I think this was a leftover from when I was updating the nomis data on move#new